ciHsc/generateDax.py

```diff
@@ -527,12 +528,13 @@ def generateDax(name="dax"):
     parser = argparse.ArgumentParser(description="Generate a DAX")
     parser.add_argument("-i", "--inputData", default="ciHsc/inputData.py",
                         help="a file including input data information")
+    parser.add_argument("-o", "--outputFile", type=str, default="ciHsc.py",
```

Comment:
I'm sure it's a typo, but the default value for the `outputFile` is misleading. The scripts spits out a file in DAX format, not a Python code. Change it to `ciHsc.dax`.